packages/azure-container-services/src/activate.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const instanaCore = require('@instana/core');
+const { backendConnector, consoleLogger } = require('@instana/serverless');
+
+const identityProvider = require('./identity_provider');
+
+const { tracing, util: coreUtil } = instanaCore;
+const { normalizeConfig } = coreUtil;
+
+let logger = consoleLogger;
+
+const config = normalizeConfig({});
+
+function init() {
+  if (process.env.INSTANA_DEBUG || process.env.INSTANA_LOG_LEVEL) {
+    logger.setLevel(process.env.INSTANA_DEBUG ? 'debug' : process.env.INSTANA_LOG_LEVEL);
+  }
+
+  instanaCore.preInit();
```

Comment:
TBD: In Fargate we use `preInit` because we loading metrics async. But here we do not have any async operations.